README.md

```diff
@@ -22,6 +22,16 @@ fast batch processing of large raster data sets.
 Please visit the **[project site](http://geotrellis.io)**
 for more information as well as some interactive demos.
 
+#### Using GeoTrellis in Other Languages
+
+If you are interested in GeoTrellis but are unable to work in Scala, then
+there is an alternative available,
+[GeoPySpark](http://github.com/locationtech-labs/geopyspark). GeoPySpark
+is a Python bindings library for GeoTrellis and can do many (but not
+all) of the operations present in GeoTrellis. While not as feature rich,
+GeoPySpark can be integrated with other tools, such as Jupyter, which
+GeoTrellis is unable to do.
```

Comment:
GeoTrellis could integrate with Jupyter through [Toree](https://toree.apache.org/) and we've prototyped it. More attractive option though is clearly for python workbook and not Scala workbook. The last clause is a little misleading but full explanation is out of place here, so I would suggest removing it.